docs/usage/configuration/config_documentation.md

```diff
@@ -2215,7 +2215,7 @@ their account.
 by the Matrix Identity Service API
 [specification](https://matrix.org/docs/spec/identity_service/latest).)
 
-*Updated in Synapse 1.64.0*: No longer accepts an `email` option.
+*Updated in Synapse 1.64.0*: The `email` option is deprecated.
```

Comment:
My editor caught all this trailing whitespace in this file - this line is the only actual change.

All right to include the whitespace trimming like this or should that be kept?

Reply:
yeah, don't worry about the whitespace changes.